utest/utils/test_normalizing.py

```diff
@@ -194,9 +194,9 @@ def test_str(self):
 
     def test_unicode(self):
         nd = NormalizedDict({'a': u'\xe4', u'\xe4': 'a'})
-        if PY2:
+        try:
             assert_equal(unicode(nd), "{'a': u'\\xe4', u'\\xe4': 'a'}")
-        else:
+        except NameError:
```

Comment:
**NO!** We definitely absolutely never ever want to have `try/except` in test code like this! It can too easily cause false positives. Also, `if PY2` makes it a lot more clear why we have two different assertions.

The whole `utest` directory should be excluded from linting altogether.